### PR TITLE
dot function for paddle.tensor.linalg

### LINKED ISSUE #16133: dot

### PR DESCRIPTION
**Pull request by benkimz**
Close #16133

### CODE REVIEW

**Review by jkeane508 — CHANGES_REQUESTED** [review]
Hello,

For this task, you also need to write a test in the appropriate location of our testing suite.

Thanks!

**Comment by benkimz** [comment]
> Hello,
> 
> For this task, you also need to write a test in the appropriate location of our testing suite.
> 
> Thanks!

Hi, I have added a test for the paddle.tensor.linalg.dot function.
Thank you

**Comment by benkimz** [comment]
> > Hello,
> > For this task, you also need to write a test in the appropriate location of our testing suite.
> > Thanks!
> 
> Hi, I have added a test for the paddle.tensor.linalg.dot function. Thank you

You can review the changes.

**Review by jkeane508 — CHANGES_REQUESTED** [review]
Hellp

It looks like tests are failing for input dimensions here - https://github.com/unifyai/ivy/actions/runs/5179793914/jobs/9333151706?pr=16148#step:3:642

We should aim to have the test you've written passing before we can merge :)

Thanks

File: ivy/functional/frontends/paddle/tensor/linalg.py [hunk 1]
```diff
@@ -6,7 +6,7 @@
     to_ivy_arrays_and_back,
 )
 
-
+# cross
```
Comment (jkeane508) [inline ivy/functional/frontends/paddle/tensor/linalg.py:9]: Im not sure why this was added to the code, but I don't think its necessary

**Review by jkeane508 — CHANGES_REQUESTED** [review]
It looks as though the test is failing here - https://github.com/unifyai/ivy/actions/runs/5195848323/jobs/9368968211?pr=16148#step:3:244

It seems you are passing the keyword "axis" when it's not a parameter.

**Pull request closed without merging**